src/openrct2-ui/windows/TrackDesignPlace.cpp

```diff
@@ -624,20 +625,19 @@ static void window_track_place_draw_mini_preview_track(
         }
 
         // Change rotation and next position based on track curvature
-        rotation &= 3;
+        curTrackRotation &= 3;
         const rct_track_coordinates* track_coordinate = &TrackCoordinates[trackType];
 
-        trackType *= 10;
```

Comment:
Is this gone intentionally?

Reply:
Yeah. I went and checked the assembly as didn't know what it was here for. This *10 was the size is the structure it's accessing. It hasn't been required for a good 2+ years.

Reply:
Ok, thanks!